src/ray/protobuf/gcs.proto

```diff
@@ -227,6 +227,12 @@ message JobTableData {
   bytes job_id = 1;
   // Whether it's dead.
   bool is_dead = 2;
+  // The UNIX timestamp corresponding to this event (job added or removed).
+  int64 timestamp = 3;
+  // IP of the node this job was started on.
+  string node_manager_address = 4;
```

Comment:
just node address?

Reply:
this is to be consistent with the ClientTable